azuredeploy.json

```diff
@@ -263,71 +318,28 @@
-                    },
                     "functionAppName": {
                         "value": "[variables('functionAppName')]"
-                    },
-                    "storageConnectionString": {
-                        "value": "[concat('DefaultEndpointsProtocol=https;AccountName=',variables('storageAccountName'),';AccountKey=',listkeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2019-06-01').keys[0].value, ';')]"
-                    },
-                    "functionProject": {
-                        "value": "[variables('functionProject')]"
-                    },
-                    "searchServiceName": {
-                        "value": "[variables('searchServiceName')]"
-                    },
-                    "qnaServiceName": {
-                        "value": "[variables('qnaServiceName')]"
-                    },
-                    "cogServicesKey": {
-                        "value": "[listkeys(resourceId('Microsoft.CognitiveServices/accounts', variables('cognitiveServicesAllInOneName')), '2017-04-18').key1]"
                     }
                 }
             }
         }
     ],
     "outputs": {
-        "Search Service Name": {
-            "type": "string",
-            "value": "[variables('searchServiceName')]"
-        },
-        "Search Service Api Key": {
-            "type": "string",
-            "value": "[listAdminKeys(resourceId('Microsoft.Search/searchServices', variables('searchServiceName')), '2020-08-01').primaryKey]"
-        },
-        "Cognitive Services AllInOne Key": {
-            "type": "string",
-            "value": "[listkeys(resourceId('Microsoft.CognitiveServices/accounts', variables('cognitiveServicesAllInOneName')), '2017-04-18').key1]"
-        },
-        "Blob Storage Account Connection String": {
-            "type": "string",
-            "value": "[concat('DefaultEndpointsProtocol=https;AccountName=',variables('storageAccountName'),';AccountKey=',listkeys(resourceId('Microsoft.Storage/storageAccounts', variables('storageAccountName')), '2019-06-01').keys[0].value, ';')]"
-        },
-        "Azure Function Name": {
-            "type": "string",
-            "value": "[variables('functionAppName')]"
-        },
-        "Azure Function Host Code": {
-            "type": "string",
-            "value": "[listkeys(concat(resourceId('Microsoft.Web/sites', variables('functionAppName')), '/host/default/'),'2019-08-01').functionKeys.default]"
-        },
         "HTTP Trigger": {
             "type": "string",
-            "value": "[concat('https://',variables('functionAppName'),'.azurewebsites.net/api/init-accelerator?code=',listkeys(concat(resourceId('Microsoft.Web/sites', variables('functionAppName')), '/host/default/'),'2019-08-01').functionKeys.default)]"
+            "value": "[reference('initaccelerator').outputs.URL.value]"
```

Comment:
we can remove this output if not needed.

Reply:
Need it for the trigger